test/eventlistener_test.go

```diff
@@ -111,10 +124,11 @@ func TestEventListenerCreate(t *testing.T) {
 		bldr.TriggerTemplate("my-triggertemplate", "",
 			bldr.TriggerTemplateSpec(
 				bldr.TriggerTemplateParam("oneparam", "", ""),
-				bldr.TriggerTemplateParam("twoparamname", "", ""),
+				bldr.TriggerTemplateParam("twopgo test -v -tags=e2e -count=1 ./test -run ^TestEventListeneraramname", "", ""),
```

Comment:
Is this intentional? As in do we expect the name (and not the value) of a template param to contain spaces and special chars?

Reply:
lol NO

Reply:
something is going on lately where i randomly paste into my IDE _all the time_. i need to get to the bottom of this XD